server/dev_router.js

```diff
@@ -37,5 +48,9 @@ router.use('/table/all', (req, res) => {
   res.json(TABLES)
 })
 
+router.use('/teamsrounds/all', (req, res) => {

```

Comment:
Please look at the issue #17 which this PR is meant to solve.
The way the matches are retrieved from the server is **by team**. This means that you request `/teams/:team_number/matches` and you get a list of the matches **for that team, and that team alone**. This is in order to save the network a lot of data transmitions.
Also notice the JSON format of the response returned to you by the `Tournament` service. This module (the `dev_server`) should be a mock-responder for these requests, and thus should accept request of the same format, and return them in the same format.
This is reflected in the client as well, because it is meant to work with the format described here (single load when the page goes up, and not every team selection), so it won't work in production